runtime/gcchk/CMakeLists.txt

```diff
@@ -29,25 +29,52 @@ j9vm_add_library(j9gcchk SHARED
 	${gcchk_sources}
 )
 
-target_link_libraries(j9gcchk
-	PRIVATE
-		j9vm_interface
-		j9util
-		j9utilcore
-		j9avl
-		j9hashtable
-		j9thr
-		j9stackmap
-		j9pool
-		j9hookable
+if(DEFINED OMR_OS_AIX AND DEFINED CMAKE_C_COMPILER_IS_OPENXL)
```

Comment:
I don't think the set of libraries should be conditional. If they're necessary for any toolchain, the dependencies should be applied to all.

Reply:
I just want to understand clearly, by all you mean all platforms? I never tested this config for zos, so to be safe I am adding them only for AIX platform.

Reply:
If the code actually depends on names available only in the libraries that were added for OpenXL, then I expect it was just wrong to omit them in the first place. Adding them only for OpenXL just continues the error. Of course, we would have to verify that the changes don't break other platforms, but I think the risk of that is very low.